src/server/renderer.js

```diff
@@ -44,14 +44,14 @@ export default (projectConfig, projectToolsConfig) => {
           console.error('ROUTER ERROR:', pretty.render(error));
           res.status(500);
         } else if (renderProps) {
-          loadPropsOnServer(renderProps, (err, asyncProps) => {
+          loadOnServer(renderProps, store).then(() => {
```

Comment:
@sars I was able to reproduce a server hang without the catch statement in there by testing my JWT example, which has a known issue (#32) getting its imports serverside on the first run of the server. Previously it would send the error (using the `err` coming back from the `loadPropsOnServer` call). But the catch call is not receiving any parameters, so it has nothing to send.